### PR TITLE
Standardize mutagen serum descriptions

### PR DESCRIPTION
**Pull request by thquinn**
#### Summary
```SUMMARY: None```

#### Purpose of change
Added descriptions for mutagen serums that didn't have them, and made some copy edits.

### CODE REVIEW

**Comment by kevingranade** [comment]
I'm in favor, but I want to let it sit for a while and accumulate copyedits and feedback since it's so subjective.

**Comment by thquinn** [comment]
> I'm in favor, but I want to let it sit for a while and accumulate copyedits and feedback since it's so subjective.

Yep, no rush.

**Comment by mon-pad** [comment]
I understand the medical mutations are more likely to be positive, but since there are still a couple of negative ones (according to the wiki), should it say "...if you really want to?" like the others?

**Comment by thquinn** [comment]
I figured the language of that one was intentionally different, in a "medical-ese" kind of way, so I left it as is. Happy to bring it in line if others agree.

File: data/json/items/comestibles/mutagen.json [hunk 1]
```diff
@@ -156,16 +161,16 @@
...
+    "color": "white",
     "use_action": { "type": "mutagen_iv", "mutation_category": "LUPINE" }
   },
   {
     "id": "iv_mutagen_medical",
     "copy-from": "iv_mutagen_flavor",
     "type": "COMESTIBLE",
     "name": "medical serum",
-    "description": "A super-concentrated substance.  Judging by the amount, it should be delivered via injection.  You'd need a syringe.",
+    "description": "A super-concentrated mutagenic substance.  Judging by the amount, it should be delivered via injection.  You'd need a syringe.",
```
Comment (ifreund) [inline data/json/items/comestibles/mutagen.json]: I would bring this in line with the others, don't see a good reason to have it be the only different one.

File: data/json/items/comestibles/mutagen.json [hunk 2]
```diff
@@ -176,7 +181,7 @@
     "copy-from": "iv_mutagen_flavor",
     "type": "COMESTIBLE",
     "name": "plant serum",
-    "description": "A super-concentrated mutagen strongly resembling tree sap.  You need a syringe to inject it...  if you really want to?",
+    "description": "A super-concentrated mutagen resembling tree sap.  You need a syringe to inject it...  if you really want to?",
```
Comment (ifreund) [inline data/json/items/comestibles/mutagen.json]: Hmm, I'm pretty sure that in it's fluid form tree sap is colorless, it's only when it thickens and coagulates to seal a wound that it takes a color.  Still, this is no more unrealistic than any of these other descriptions really.

**Comment by thquinn** [comment]
Addressed comments.

File: data/json/items/comestibles/mutagen.json (hunk 2 above)
Comment (thquinn) [reply data/json/items/comestibles/mutagen.json]: I suppose that's true. I referenced amber in another of the descriptions, so I'll just change it to something else for variety's sake.

**Pull request merged**